Source/Core/DolphinQt/HotkeyScheduler.cpp

```diff
@@ -162,7 +163,8 @@ void HotkeyScheduler::Run()
     if (Core::GetState(Core::System::GetInstance()) != Core::State::Stopping)
     {
       // Obey window focus (config permitting) before checking hotkeys.
-      Core::UpdateInputGate(Config::Get(Config::MAIN_FOCUSED_HOTKEYS));
+      const bool is_tas_input_focused = Host::GetInstance()->GetTASInputFocus();
+      Core::UpdateInputGate(Config::Get(Config::MAIN_FOCUSED_HOTKEYS) && !is_tas_input_focused);
```

Comment:
UpdateInputGate also checks Host_TASInputHasFocus. Do we need the call to GetTASInputFocus here?

Reply:
No we don't, removed.